pkg/configauditreport/controller.go

```diff
@@ -191,7 +193,11 @@ func (r *ResourceController) reconcileResource(resourceKind kube.Kind) reconcile
 		}
 
 		// Skip processing if there are no policies applicable to the resource
-		applicable, reason, err := policies.Applicable(resource)
+		c, err := r.NewConfigForConfigAudit(r.PluginContext)
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		applicable, reason, err := policies.Applicable(resource, c.GetSupportedConfigAuditKinds())
```

Comment:
Doesn't it make more sense pass `c.GetSupportedConfigAuditKinds()` should be passed to `NewPolicies`?